app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SubscriptionController.scala

```diff
@@ -96,12 +98,14 @@ class SubscriptionController @Inject()
     implicit request =>
       hasEnrolments match {
         case true => Future(Redirect(routes.CheckAgencyController.showHasOtherEnrolments()))
-        case false => sessionStoreService.fetchKnownFactsResult.map(_.map { knownFactsResult =>
-          Ok(html.subscription_details(knownFactsResult.taxpayerName, initialDetailsForm.fill(
-            InitialDetails(knownFactsResult.utr, knownFactsResult.postcode, null, null, null))))
-        }.getOrElse {
-          sessionMissingRedirect()
-        })
+        case false =>
+          mark("Count-Subscription-CleanCreds-Success")
```

Comment:
why only mark() false?, mark "true" above and the "getOrElse" below also?

Reply:
we are only interested in key steps, not in all events